File: array/674.py
class Solution:
    def findLengthOfLCIS(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        if not nums:
            return 0
        ret = 1
        cur = 1
        for i in range(0, len(nums) - 1):
            if nums[i] < nums[i+1]:
                cur += 1
            else:
                cur = 1
            if cur > ret:
                ret = cur
                
        return ret

File: array/674.go
func findLengthOfLCIS(nums []int) int {
    if len(nums) == 0 {
        return 0
    }
    ret := 1
    cur := 1
    for i := 1; i < len(nums); i++ {
        if nums[i] > nums[i-1] {
            cur++
        }else {
            cur = 1
        }
        if ret < cur {
            ret = cur
        }
    }
    return ret
}
